source/data/transfer/filezilla.rst

```diff
@@ -4,59 +4,76 @@
 Data transfer with FileZilla
 ############################
 
-FileZilla is an easy-to-use freely available ftp-style program to
-transfer files to and from your account on the clusters.
+FileZilla is a free and easy-to-use FTP-style program to
+transfer files to and from your VSC account on the clusters.
 
 You can also put FileZilla with your private key on a USB stick to
 access your files from any internet-connected PC.
```

Comment:
As this assumes key-based authentication it should also be addressed in this PR, and I think the best is to just remove this altogether. If this approach needs to be advertised (which I doubt), it would rather belong in pages like "Security Keys" or "Access from multiple computers" (but so I don't recommend adding it there either).
```suggestion
```